sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2024-2025.bpmn

```diff
@@ -652,532 +829,344 @@ dmnX (comes from dmn)</bpmn:text>
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=if &#10;studentDataCurrentYearIncome = null&#10;then&#10;	if &#10;    studentDataCRAReportedIncome = null&#10;	then&#10;		calculatedDataTaxReturnIncome&#10;	else&#10;		studentDataCRAReportedIncome&#10;else&#10;studentDataCurrentYearIncome" target="calculatedDataStudentTotalIncome" />
+          <zeebe:output source="=if (calculatedDataTotalAssessedNeed &#62;= 1)&#10;  and (calculatedDataTotalFamilyIncome &#60; dmnPartTimeAwardFamilySizeVariables.limitAwardCSLPThresholdIncome)&#10;then true&#10;else false" target="awardEligibilityCSLP" />
         </zeebe:ioMapping>
       </bpmn:extensionElements>
-      <bpmn:incoming>Flow_0a0yhpv</bpmn:incoming>
-      <bpmn:outgoing>Flow_0dqxcli</bpmn:outgoing>
+      <bpmn:incoming>Flow_1vkmf8w</bpmn:incoming>
+      <bpmn:outgoing>Flow_0kra8re</bpmn:outgoing>
     </bpmn:intermediateThrowEvent>
-    <bpmn:intermediateThrowEvent id="Event_12o98iq" name="Calculate Eligible Dependents for Child Care Cost">
+    <bpmn:sequenceFlow id="Flow_0uvmsmj" sourceRef="Event_0tjlor6" targetRef="Gateway_1uj6ty4" />
+    <bpmn:exclusiveGateway id="Gateway_14myy0o" default="Flow_1mtcoea">
+      <bpmn:incoming>Flow_1dkitnr</bpmn:incoming>
+      <bpmn:outgoing>Flow_1mtcoea</bpmn:outgoing>
+      <bpmn:outgoing>Flow_0btwcpj</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:intermediateThrowEvent id="Event_0tjlor6" name="Define minimum transport weekly limit">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=count(calculatedDataDependants[&#10;      // Dependant who is between 0 and 11 years on study start date.&#10;      // i.e. until the day(study start date) before their 12th birthday the dependant &#10;      // is considered to be in this category.&#10;      date(item.dateOfBirth) &#62; (date(offeringStudyStartDate) - duration(&#34;P12Y&#34;))]&#10;  )" target="calculatedDataDependants11YearsOrUnder" />
-          <zeebe:output source="=count(calculatedDataDependants[&#10;      // Dependant who is 12 years or older on study start date.&#10;      // i.e. from the day(study start date) starting their 12th birthday the dependant &#10;      // is considered to be in this category.&#10;      date(item.dateOfBirth) &#60;= (date(offeringStudyStartDate) - duration(&#34;P12Y&#34;))&#10;      and item.declaredOnTaxes = &#34;yes&#34;]&#10;  )" target="calculatedDataDependants12YearsOverOnTaxes" />
-          <zeebe:output source="=calculatedDataDependants11YearsOrUnder + calculatedDataDependants12YearsOverOnTaxes" target="calculatedDataTotalEligibleDependentsForChildCare" />
+          <zeebe:output source="=//If student is in onsite courses they get a minimum per study week, otherwise 0&#10;if(offeringDelivered = &#34;onsite&#34;)&#10;then &#10; dmnPartTimeProgramYearMaximums.limitTransportationAllowance&#10;else&#10;  0" target="calculatedDataWeeklyMinTransport" />
```

Comment:
Pretty minor but some comments have spaces after the "//" and some have a period or not. Maybe from this PR or from prior ones. No need to change, just double-checking if the below is what we follow.
```
// Some comment.
```